### PR TITLE
MCLOUD-6524: MC-35514: Can't create shipping label for existing order all JS components in sliding modal doesn't work.

### PR DESCRIPTION
**Pull request by viktorpetryk**
### Description

https://jira.corp.magento.com/browse/MCLOUD-6524
Fix for issue with creating shipping label for existing order.

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MC-35514

### Manual testing scenarios

1. ...
2. ...

### Release notes

For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud Patches release notes](https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html).

### Associated documentation updates

Add link to Magento DevDocs PR or Issue, if needed.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] Pull request introduces user-facing changes and includes meaningful updates for any required release notes and documentation changes
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)

### CODE REVIEW

File: patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch [hunk 1]
```diff
@@ -0,0 +1,244 @@
+diff --git a/vendor/magento/module-shipping/Block/Adminhtml/Order/Packaging.php b/vendor/magento/module-shipping/Block/Adminhtml/Order/Packaging.php
```
Comment (oshmyheliuk) [inline patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch]: Please change `diff --git` to `diff -Nuar` as all patches have such format.

File: patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+index e5e419328eea..827b26c64b33 100644
```
Comment (oshmyheliuk) [inline patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch]: Please remove this and all index lines as they are redundant.

File: patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch (hunk 1 above)
Comment (viktorpetryk) [reply patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch]: Fixed.

File: patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch (hunk 2 above)
Comment (viktorpetryk) [reply patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch]: Fixed.

**Comment by victorpetryk** [comment]
@magento run all tests

**Comment by viktorpetryk** [comment]
After last commit with fixes there is a one failed build https://travis-ci.com/github/magento/magento-cloud-patches/jobs/363624493.
@oshmyheliuk Could you advise how to rerun it or it may be related to last changes?

**Comment by oshmyheliuk** [comment]
@victorpetryk failed build isn't related to changes, looks like some general error, I've restarted failed job

**Review by oshmyheliuk — APPROVED** [review]

File: patches.json [hunk 3]
```diff
@@ -252,6 +252,9 @@
     },
     "Large amount of Keys Block Cache": {
       ">=2.3.1 <2.3.5": "MDVA-22950__large_amount_of_keys_block_cache__2.3.1.patch"
+    },
+    "Fix shipping label creation issue": {
+     "2.4.0": "MC-35514__fix_shipping_label_creation_issue__2.4.0.patch"
```
Comment (shiftedreality) [inline patches.json:257]: @victorpetryk This won't be applied for any of `2.4.0` patch versions, is it expected?

Comment (viktorpetryk) [reply patches.json:257]: @shiftedreality As I know, actual fix will be delivered to 2.4.1 and there will be no need for a patch.

**Review by shiftedreality — APPROVED** [review]

**Comment by viktorpetryk** [comment]
During working on the task https://jira.corp.magento.com/browse/MC-35514 to deliver fix for 2.4.1 we found couple of issues that need to be fixed.
@oshmyheliuk, @shiftedreality, @mveeramneni Please don't deliver this PR until patch will be updated.
And, please, advise do I need to close the current PR and create another one with updated patch or just merge new fixes to this one.

**Comment by shiftedreality** [comment]
Closing, please create new once ready

**Comment by shiftedreality** [comment]
Since it's just patches, you actually may update this PR.

Adding on-hold label

**Pull request closed without merging**